mk/sm.spec.in

```diff
@@ -285,8 +288,10 @@ cp -f /etc/lvm/lvm.conf.orig /etc/lvm/lvm.conf || exit $?
 %config /etc/udev/rules.d/40-multipath.rules
 %config /etc/udev/rules.d/55-xs-mpath-scsidev.rules
 %config /etc/udev/rules.d/62-multipath.rules
+%config /etc/udev/rules.d/58-xapi.rules
```

Comment:
add another rules for 69-dm-lvm-metad.rules
%config /etc/udev/rules.d/69-dm-lvm-metad.rules